apps/frontend/src/app/PageArtifact/ArtifactEditor.tsx

```diff
@@ -134,7 +134,6 @@ export type ArtifactEditorProps = {
   allowUpload?: boolean
   allowEmpty?: boolean
   disableSet?: boolean
-  disableSlot?: boolean
   fixedSlotKey?: ArtifactSlotKey | ''
```

Comment:
Can we make it so that the type definition is simply `fixedSlotKey?: ArtifactSlotKey`? I want to move away from the `''` as falsy. This will also clean up some of the `fixedSlotKey !== ''` code as well

Reply:
Oh, yeah. We always default to flower anyway for the dropdown so this is probably kinda redundant too.

Reply:
done